lib/devices.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  328: {name: "SensorInOne", class: "sensor"},
+  4000: {name: "SMS Module", class: "extension"},
+  621: {name: "6 channel 1A Dimmer", class: "dimmer"},
+  172: {name: "DLP panel", class: "switch"},
+  211: {name: "6 channel floor heating", class: "heating"},
+  42: {name: "Dali Module", class: "extension"},
+  705: {name: "2 channel curtain controller", class: "curtain"},
+  706: {name: "2 channel curtain controller", class: "curtain"},
+  1109: {name: "Logic controller", class: "extension"},
+  445: {name: "8 channel relay", class: "relay", channels: 8, commands: {status: 0x0033}},
```

Comment:
I'll rewrite this to only include name and class for now, and add it to `device.js`. I would say it makes sense to return the number of channels available if we have a list of HDL models, and have requested such a list from the local HDL office. In addition it would be nice to know what kind of capabilities the model has, such as "dim" or "onoff" But we can extend this in a later pull request.

Reply:
Sure it would be nice to know capabilities of devices, but it should not be hardcoded here.

This small library only provides implementation of messaging protocol used by HDL and doesn't require updates when HDL releases new devices. 

If HDL devices has special message with description of its capabilities this library could parse it as any other command message. 

Actually  `Device` and `Channel` classes is redundant here and has been created only as convenient DSL for ad-hoc scripts. For simple interaction with HDL bus everything what you need is a `Bus` instance.

For example if you are using some abstract tool like `node-red` you will just need an interface to send messages (instance of `Bus` class). Or if you have your own DSL for IoT you can wrap `Bus.send` with it.